packages/hooks/src/hooks/useTokenBalance/useTokenBalance.ts

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { ethers } from 'ethers';
+import { Web3Context } from '../../Provider';
+import ERC20 from './ERC20.json';
+
+export const useTokenBalance = (address: string, owner: string) => {
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [balance, setBalance] = React.useState<null | string>(null);
```

Comment:
This should be fine, and will default `balance` to `undefined`
```suggestion
  const [balance, setBalance] = React.useState<string>();
```